src/config/index.js

```diff
@@ -34,6 +34,8 @@ import { contentIcons } from './ContentIcons';
 
 import applyAddonConfiguration from 'load-volto-addons';
```

Comment:
@tiberiuichim since this is no longer true, shouldn't we rename it to `applyConfig` or so?